spark/src/main/scala/io/delta/sharing/spark/model.scala

```diff
@@ -75,14 +89,65 @@ private[sharing] case class Protocol(minReaderVersion: Int) extends Action {
   override def wrap: SingleAction = SingleAction(protocol = this)
 }
 
+sealed abstract class AddFileBase(
+    url: String,
+    id: String,
+    @JsonInclude(JsonInclude.Include.ALWAYS)
+    partitionValues: Map[String, String],
+    size: Long,
+    @JsonRawValue
+    stats: String = null)
+    extends Action {}
```

Comment:
There is some conflict here with changes in my PR:
https://github.com/delta-io/delta-sharing/pull/149

Perhaps wait for that PR before we merge this one?

Reply:
sg, will do